05-JWT-Basics/starter/middleware/auth.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require("jsonwebtoken");
+const {UnauthenticatedError} = require("../errors");
+
+const authenticationMiddleware = async (req, res, next) => {
+  const authHeader = req.headers.authorization;
+  //console.log(req.headers.authorization)
+
+  if (!authHeader || !authHeader.startsWith("Bearer")) {
```

Comment:
extra challenge: can we rewrite this handling of the authorization header to use a simple regular expression?